inc/com/centreon/engine/contact.hh

```diff
@@ -127,6 +121,20 @@ class                           contact {
                                 get_service_notification_commands() const;
   std::list<std::shared_ptr<commands::command>>&
                                 get_service_notification_commands();
+  std::list<std::shared_ptr<contactgroup>> const&
+                                get_parent_groups() const;
+  std::list<std::shared_ptr<contactgroup>>&
```

Comment:
there is a contactgroup_map struct in contactgroup.h, Is the std::list more appropriate ?